packages/arb-token-bridge-ui/src/pages/_app.tsx

```diff
@@ -50,10 +50,27 @@ Sentry.init({
     /^WebSocket connection failed for host: wss:\/\/relay.walletconnect.org$/i
   ],
   beforeSend: (event, hint) => {
+    if (!hint.originalException) {
+      return event
+    }
+
     if (isUserRejectedError(hint.originalException)) {
       return null
     }
 
+    const { code, message } = hint.originalException as {
+      code?: number
+      message?: string
+    }
+
+    if (code && message) {
+      Sentry.getCurrentScope().setFingerprint([
+        '{{ default }}',
+        code.toString(),
+        message
+      ])
```

Comment:
Current issue have multiple unrelated events, this should help split the future events properly